components/CollaboratorsAmount/SubmitForm.tsx

```diff
@@ -7,23 +8,37 @@ const SubmitForm = () => {
   const { setTab, collaboratorsAmount, setCollaboratorsAmount } =
     useContractBuilderProvider()
 
+  const handleCollaboratorsChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value
+
+    if (value === '') {
+      setCollaboratorsAmount(0)
+
+      return
+    }
+
+    if (isNaN(parseInt(e.target.value, 10))) return
+
+    setCollaboratorsAmount(parseInt(e.target.value, 10))
+  }
+
   return (
     <>
       <div className=" size-full flex flex-col md:justify-center justify-start z-[1]">
-      <div className="hidden md:block">
+        <div className="hidden md:block">
           <PassedQuestions />
         </div>
         <p className="text-white md:text-3xl tracking-[-0.05rem] font-share pt-6 text-[20px]">
           How many collaborators contributed to writing the song?
         </p>
-      
-         <label htmlFor="#songName" className="mt-6 text-[#696969] text-[15px] ">
+
+        <label htmlFor="#songName" className="mt-6 text-[#696969] text-[15px] ">
           <input
             id="songName"
             type="text"
             className="  !outline-none font-rubik rounded-md border-white border !bg-transparent mt-6 p-2 max-w-[90px]"
-          placeholder="2"
-            onChange={(e) =>  setCollaboratorsAmount(parseInt(e.target.value, 10))}
+            placeholder="2"
+            onChange={handleCollaoratorsAmount}
```

Comment:
Did you test your changes???
- I'm not sure this function is defined anywhere...